# Exception handler function

require `java.lang.RuntimeException`
require `java.io.IOException`

handler(e) =>
  RuntimeException        : e.getClass().name
  IOException             : 'some worries'
  *                       : e


main except handler ->
  `loop.Loop`.error('test')
